tests/gui/test_web.py

```diff
@@ -72,6 +81,37 @@ def test_release_checker():
     assert checker.releases[1] != rls_test
 
 
+def test_announcementchecker():
+
+    BULLETIN_JSON_PATH = "D:\\TalmoLab\\sleap\\tests\\data\\announcement_checker_bulletin\\test_bulletin.json"
```

Comment:
Use relative paths so that others can run your code as well. Also, I don't think this is included in the fixtures/repo?